proto/gnmi_ext/gnmi_ext.proto

```diff
@@ -73,3 +74,18 @@ message Role {
   // More fields can be added if needed, for example, to specify what paths the
   // role can read/write.
 }
+
+// The History extension allows clients to request historical data. Its
+// spec can be found at
+// https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-history.md
+message History {
+  oneof request {
+    int64 time = 1;
+    TimeRange range = 2;
+  }
+}
+
+message TimeRange {
+  int64 start = 1;
```

Comment:
(nit): Add a short comment just explaining that these are both in nanoseconds since the epoch.